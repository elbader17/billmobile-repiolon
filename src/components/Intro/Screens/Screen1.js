import React, { Component } from 'react';
import { Text, View } from 'react-native';
import style from './style';
import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';

class Screen1 extends Component {
  render() {
    return (
      <KeyboardAwareScrollView>
        <View style={style.silde}>
          <View style={style.centerVertical}>
            <Text style={style.number}>1</Text>
            <Text style={style.textTittle}>Facturar sin demoras</Text>
            <Text style={style.textDescription}>Como cuando lo era cuando lo hacias en papel, 
            Ã¡gil para vos y para tus clientes.</Text>
            </View>
        </View>
      </KeyboardAwareScrollView>
    );
  }
}

export default Screen1;