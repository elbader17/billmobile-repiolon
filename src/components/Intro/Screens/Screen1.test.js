import React from 'react';
import renderer from 'react-test-renderer';
import Screen1 from './Screen1';

describe('Rendering Component Screen1', () => {
  test('renders correctly', () => {
    const component = renderer.create(<Screen1 />).toJSON();
    expect(component).toMatchSnapshot();
  });
})
