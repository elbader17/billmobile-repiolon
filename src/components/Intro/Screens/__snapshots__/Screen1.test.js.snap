// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Component Screen1 renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3687d1",
      "height": 1334,
      "justifyContent": "center",
      "width": 750,
    }
  }
>
  <Text
    style={
      Object {
        "bottom": 422,
        "color": "#fff",
        "fontFamily": undefined,
        "fontSize": 80,
        "lineHeight": 90,
        "position": "absolute",
      }
    }
  >
    1
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "fontFamily": undefined,
        "fontSize": 16,
        "height": 33,
        "lineHeight": 32,
        "position": "absolute",
        "textAlign": "center",
        "top": 287,
        "width": 289,
      }
    }
  >
    Facturar sin demoras
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "fontFamily": undefined,
        "fontSize": undefined,
        "height": 66,
        "lineHeight": 21,
        "position": "absolute",
        "textAlign": "center",
        "top": 332,
        "width": 289,
      }
    }
  >
    Como cuando lo era cuando lo hacias en papel, Ã¡gil para vos y para tus clientes.
  </Text>
</View>
`;
